ydb/_topic_reader/topic_reader_asyncio.py

```diff
@@ -174,6 +181,14 @@ async def _connection_loop(self):
                 await asyncio.sleep(retry_info.sleep_timeout_seconds)
 
                 attempt += 1
+            finally:
+                if self._stream_reader is not None:
+                    # noinspection PyBroadException
+                    try:
+                        await self._stream_reader.close()
+                    except BaseException:
+                        # supress any error on close stream reader
```

Comment:
you may use https://docs.python.org/3.5/library/contextlib.html#contextlib.suppress